### PR TITLE
Proxify

### LINKED ISSUE #106: Wrap client as FMU
As suggested in #71 , it would be nice if we were able to wrap one of the clients as an FMU.
This would allow distributed simulation to any FMI compliant tool without modification.

Obvious problem: The `modelDescription.xml` refers to a specific model. 
Solution: Do not provide a `modelDescription.xml`. Let the user copy it from the target FMU and paste it into the archive we provide. 

Similarly the IP and port could be put into a .txt file inside the archive.

### PR DESCRIPTION
**Pull request by markaren**
This PR allows FMUs (1.0 & 2.0) to be wrapped inside a proxy FMU compatible with FMU 2.0, powered by FMI4j.

CLI usage:

```
Usage: fmu-proxify [-h] [-d=<destFile>] -f=<fmuFile> [-r=<remoteAddress>]
  -d, --dest=<destFile>   Where to save the FMU.
  -f, --file=<fmuFile>    Path to the FMU to proxify.
  -h, --help              Print this message and quits.
  -r, --remote=<remoteAddress>
                          Optional host to connect to. e.g. 127.0.0.1:9090
```

Superseeds PR #116 
Closes #106

### CODE REVIEW

**Comment by markaren** [comment]
@chrbertsch I have finally gotten around to implementing your suggestion of creating an FMU out of this project.
The PR probably needs some more polishing, but the basics works.

The generated FMU is  also capable of wrapping FMI 1.0 models as FMI 2.0 ones.
It also starts a process for each FMU instance loaded, so FMUs with the `canOnlyBeInstantiatedOnce=true` flag can be instantiated as many times as you want.  
The FMU embeds it's own server, so when running on localhost, the user don't need to start anything manually. However, using a text file inside the FMU you can override this behavior and point it to a remote server on some other computer.

Basically, with this PR users should no longer implement their own RPC clients. Simply use it as a regular FMU.

The FMUs only requirement is JDK8+ (Perhaps even a JRE is enough).

**Comment by markaren** [comment]
It does not work on Linux. No idea why..

**Comment by markaren** [comment]
Great! It now works on linux also
